packages/fxa-settings/src/components/App/RememberPassword/index.test.tsx

```diff
@@ -0,0 +1,56 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import 'mutationobserver-shim';
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+import { act, fireEvent, render, screen } from '@testing-library/react';
+import { HomePath } from '../../../constants';
+import RememberPassword from '.';
+
+jest.mock('../../../lib/metrics', () => ({
+  logViewEvent: jest.fn(),
+  settingsViewName: 'quuz',
+}));
+
+const mockNavigate = jest.fn();
+jest.mock('@reach/router', () => ({
+  ...jest.requireActual('@reach/router'),
+  useNavigate: () => mockNavigate,
+}));
+
+const mockEmail = 'test@test.com';
+
+describe('RememberPassword', () => {
+  it('renders', async () => {
+    render(<RememberPassword />);
+
+    expect(
+      screen.getByTestId('remember-password-container')
+    ).toBeInTheDocument();
+    expect(screen.getByTestId('remember-password-link')).toBeInTheDocument();
```

Comment:
Same comments here and in other files about the async function + preferably not using test IDs.